app/src/main/java/com/usepace/android/messagingcenter/utils/AudioRecorderUtil.java

```diff
@@ -0,0 +1,59 @@
+package com.usepace.android.messagingcenter.utils;
+
+import android.content.Context;
+import android.media.MediaRecorder;
+import android.util.Log;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Created by mohammednabil on 2019-07-29.
+ */
+public class AudioRecorderUtil {
+
+    private static AudioRecorderUtil audioRecorderUtil = null;
+    private MediaRecorder recorder = null;
+    private String fileName = null;
+
+    public void startRecording(Context context) {
+        fileName = context.getExternalCacheDir().getAbsolutePath() + "record_" + System.currentTimeMillis() + ".wav";
```

Comment:
We should record in mp4 instead of .wav. .wav size will be much greater than .mp4

Reply:
The Actual record is .3gp 

but the extension is .wav so it won't be readable by device media apps

Reply:
yes you are right, but the least sized item is 3gp, with .wav format iOS will be able to read it ?

Reply:
yes they can be able to play it. Let us keep it as it is then and release a version. when iOS guys implement it we will see and change accordingly if needed.